pkgs/build-support/dart/build-dart-application/default.nix

```diff
@@ -17,7 +17,22 @@
 , sourceRoot ? "source"
 , packageRoot ? (lib.removePrefix "/" (lib.removePrefix "source" sourceRoot))
 , gitHashes ? { }
-, sdkSourceBuilders ? { }
+, sdkSourceBuilders ? {
```

Comment:
Wouldn't it be better to change the line where `sdkSourceBuilders` is inherited and do `{ "dart" = ... } // sdkSourceBuilders` instead?

Reply:
Could do that, I didn't do that to be able to remove the `dart` attribute. But I don't think there's a situation where that would be useful, right?

Reply:
Yeah my suggestion allows for removing the dart attribute since it's the base attr set which you merge the variable onto. So not specifying dart would keep the default but overriding dart does override it.